CopilotKit/CopilotKit
# feat: inject minified css in bundle

# Description

- [x] removes the need to import `styles.css` manually
- [x] empty `styles.css` included in the build for backwards compatibility
- [x] uses tsup's `injectStyles` with `postcss` to bundle and minify the CSS, then inject it as a style tag
- [x] currently uses my fork of `tsup` where I added support for async function in `injectStyles` (must-have for postcss), a PR from my fork to the main library will follow shortly
- [x] remove material-ui, and use `react-icons` for icons (same icons as before)
- [x] remove unused `IncludedFilesPreview` component

# How to test this

Check out the preview deployment below.
- To test the changes of `react-ui`, head over to `/presentation`
- To test the changes of `react-textarea`, head over to `/textarea`
